decidim-proposals/app/commands/decidim/proposals/admin/publish_participatory_text.rb

```diff
@@ -37,20 +38,28 @@ def call
         attr_reader :form
         attr_reader :publish_failures
 
-        def resort_proposals(form)
+        def update_contents_and_resort_proposals(form)
```

Comment:
You don't need to pass the `form` attribute since you can already access it thanks to the `attr_reader`

Reply:
Yes I know, but I like it this way as it is more explicit. Is it a problem?